spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -154,8 +154,12 @@ def __init__(self, config, spot: Spot, stopwatch=None):
                     len(self.detections_buffer["detections"]) > 0
                 ), "Mask R-CNN msgs not found!"
                 rospy.loginfo(f"[{self.node_name}]: ...msgs received.")
+
+                # TODO: Scale pub is not being used anywhere.. remove!
```

Comment:
What is this?

Reply:
There was a scale publisher in the code which should be publishing the scale of the image. But there is no subscriber for this. Thus the publisher is redundant.